show/main.py

```diff
@@ -7,6 +7,8 @@
 import os
 import subprocess
 import sys
+import netifaces
+from netaddr import *
```

Comment:
It appears you're only using `IPAddress` from netaddr. I feel you should do one of the following instead;

`import netaddr`, then call `netaddr.IPAddress()` throughout the file

or,

`from netaddr import IPAddress`

I lean toward the former.

Reply:
Done